### PR TITLE
KukaJAVAdriver construct() initializes StartArm mode

### PR DESCRIPTION
**Pull request by ahundt**
Adds StartArm as an option in the Java code, initializes it in the Java Driver, and just does a hold postion command and prints that the connection is established.

### CODE REVIEW

File: include/grl/ros/KukaLBRiiwaROSPlugin.hpp [hunk 1]
```diff
@@ -330,13 +330,15 @@ namespace grl {
...
              if (debug) {
-              ROS_INFO("Sending start!");
+               ROS_INFO("Sending StartArm!");
              }
              break;
            case grl::flatbuffer::ArmState_ShutdownArm:
              break;
            default:
-             ROS_INFO("KukaLBRiiwaROSPlugin in unsupported mode!");
+             if(debug) {
```
Comment (ahundt) [inline include/grl/ros/KukaLBRiiwaROSPlugin.hpp]: maybe I shouldn't do this now? or maybe add a modulo counter so it prints only occasionally?

Comment (tdinesh) [inline include/grl/ros/KukaLBRiiwaROSPlugin.hpp]: modulo counter sounds better. Also, looks like you have the emergency stop #123 included in this pull request. Not sure if you want a separte merge

Comment (ahundt) [inline include/grl/ros/KukaLBRiiwaROSPlugin.hpp]: meh it is small enough that if it works I'll just leave the estop check in.

Comment (tdinesh) [inline include/grl/ros/KukaLBRiiwaROSPlugin.hpp]: I will test this tomorrow and get back. Thanks!

Comment (ahundt) [inline include/grl/ros/KukaLBRiiwaROSPlugin.hpp]: added the modulo counter

**Comment by tdinesh** [comment]
+1 Looks good. The smartpad gets flooded with `"StartArm mode active, connection establ" `
Maybe do % 10000? Given 1k Hz loop rate the messages currently spit twice every sec.

**Comment by tdinesh** [comment]
Or Maybe only display once when a mode change is requested?

**Comment by ahundt** [comment]
cool I'll merge this then!

I think the 10k you suggested is good. Those settings should print on mode changes and an occasional "I'm alive" printout.

**Pull request merged**